Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,118 @@
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  if [[ "$HOMEBREW_PROCESSOR" = "Intel" ]]
+  then
+    ruby_URL="https://github.com/xu-cheng/homebrew-portable/releases/download/20160512/portable-ruby-2.0.0-p648.x86_64-darwin9.tar.gz"
```

Comment:
I didn't save the 32bit build yet. I can build one if you needed. But why cannot we use 64bit version?